examples/security/webserver-signatures/src/main/resources/service1.yaml

```diff
@@ -57,16 +59,17 @@ security:
             signature:
               key-id: "service1-rsa"
               private-key:
-                # path to keystore
-                keystore-path: "src/main/resources/keystore.p12"
-                # Keystore type
-                # PKCS12, JSK or RSA (not really a keystore, but directly the linux style private key unencrypted)
-                # defaults to jdk default
-                keystore-type: "PKCS12"
-                # password of the keystore
-                keystore-passphrase: "password"
-                # alias of the key to sign request
-                key-alias: "myPrivateKey"
+                keystore:
+                  # path to keystore
+                  resource.path: "src/main/resources/keystore.p12"
+                  # Keystore type
+                  # PKCS12, JSK or RSA (not really a keystore, but directly the linux style private key unencrypted)
+                  # defaults to jdk default
+                  type: "PKCS12"
+                  # password of the keystore
+                  passphrase: "password"
+                  # alias of the key to sign request
+                  key.alias: "myPrivateKey"
```

Comment:
Yaml at its best

Reply:
This was actually quite a big change in resource configuration. Both approaches are still supported, the original one is deprecated.